### ISSUE TITLE
Option request: "--timeout=300s" for "create"

### ISSUE THREAD

**Issue by bsideup** [cloudfoundry-incubator/lattice#56]
"app took too long to start.".

Probably because of big image downloading from registry

**Comment by dhilipkumars** [comment]
Hi @bsideup 

Have tried to use LATTICE_CLI_TIMEOUT environment variable?

as explained in the below link? 
http://lattice.cf/docs/troubleshooting.html

Dhilip

**Comment by bsideup** [comment]
Hi @dhilipkumars 

No, I didn't. But it feels little bit dirty to use env vars for it.

I will try, thanks!

**Comment by dhilipkumars** [comment]
Hi @bsideup 

From what i understand lattice comamnds are async in nature.  You request for a desired state and by default wait for 60secs.   (can be overridden by he above mentioned environment variable) 

i generally monitor the system with

```
$ltc debug-logs
```

**Comment by davidwadden** [comment]
@bsideup We'll discuss whether we want to implement a --timeout flag for the commands that poll the system.  For now, the `LATTICE_CLI_TIMEOUT` (in seconds) environment variable should perform the same functionality.  e.g.,

```bash
$ LATTICE_CLI_TIMEOUT=60 ltc create lattice cloudfoundry/lattice-app
```

Also, even if you get the message "app took too long to start",  the system is still attempting to start the app -- you can check with either `ltc list` or `ltc status <app>`.

**Comment by onsi** [comment]
we're going to do the following:
1. double the default timeout
2. add `--timeout` in lieu of the environment variable
3. improve the error message to explain that the container did not actually fail to create, only that `ltc` has stopped waiting for it.  In the background, Lattice continues to download the layers.

**Comment by bsideup** [comment]
@onsi great news :) much more user friendly IMO

**Comment by davidwadden** [comment]
these changes are included with the v0.2.1-48-gd66edad unstable build

**Comment by onsi** [comment]
This all just landed in v0.2.2
